x-pack/legacy/plugins/apm/public/components/app/Settings/CustomizeUI/CustomActions/index.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiPanel, EuiSpacer } from '@elastic/eui';
+import { isEmpty } from 'lodash';
+import React, { useState } from 'react';
+import { ManagedTable } from '../../../../shared/ManagedTable';
+import { Title } from './Title';
+import { EmptyPrompt } from './EmptyPrompt';
+import { Flyout } from './Flyout';
+
+export const CustomActions = () => {
```

Comment:
maybe `CustomActionsTable` is more appropriate here?

Reply:
hmmm, I don't know, this is the `Custom Actions` page we don't call the `Service` and `Trace` pages `ServiceTable` and `TraceTable`, even though they have a table. WDYT about `CustomActionsOverview`?

Reply:
CustomActionsOverview sounds good 👍